modules/gridBidAdapter.js

```diff
@@ -89,6 +90,14 @@ export const spec = {
           content = jwTargeting.content;
         }
       }
+      if (ortb2) {
```

Comment:
I believe ortb2 is always null here - am I missing something? AFAIK the `bid` object does not have that property.

Reply:
OK I understand now - one particular RTD provider adds that to the bids (https://github.com/prebid/Prebid.js/pull/7886). That does not make much sense to me, why would it demand an update to every bid adapter instead of calling `setConfig`?

Reply:
fwiw #7886 caused very large revenue loss to cafemedia and we upgraded to prebid 6.7 with it carved out. we are still investigating why

Reply:
We're actually going to be super happy if the JWP RTD module can move from bid object to the regular ortb2 approach so that our standard integration would work for this

Reply:
Hi! We will correct this ASAP, thank you for raising!

Reply:
@patmmccann @karimMourra I guess, as soon as JW team will support upgrading their RTD module to setConfig, this part of the MediaGrid PR won't be needed anymore, right? Let's make sure the two PRs fall into the same release

Reply:
@TheMediaGrid  please note that https://github.com/prebid/Prebid.js/pull/8338#issuecomment-1112999837 is migrating away from reading ortb2 from getConfig

Reply:
PR for JW RTD: https://github.com/prebid/Prebid.js/pull/8354

Reply:
@patmmccann agreed, this one is no longer needed with 8354 going live